cmake/3rdParty/Platform/Linux/BuiltInPackages_linux_aarch64.cmake

```diff
@@ -18,7 +18,6 @@ ly_associate_package(PACKAGE_NAME xxhash-0.7.4-rev1-multiplatform
 # platform-specific:
 ly_associate_package(PACKAGE_NAME expat-2.4.2-rev2-linux-aarch64                             TARGETS expat                       PACKAGE_HASH 934a535c1492d11906789d7ddf105b1a530cf8d8fb126063ffde16c5caeb0179)
 ly_associate_package(PACKAGE_NAME assimp-5.2.5-rev1-linux-aarch64                            TARGETS assimplib                   PACKAGE_HASH 0e497e129f9868088c81891e87b778894c12486b039a5b7bd8a47267275b640f)
-ly_associate_package(PACKAGE_NAME AWSNativeSDK-1.9.50-rev3-linux-aarch64                     TARGETS AWSNativeSDK                PACKAGE_HASH d03fc208ff2fc08b18568f3b6f072cb0de7605ab1dd314118b175f3f67aa04b3)
```

Comment:
Why is the AWS SDK for C++ association being removed?

Reply:
whoops